apps/server/src/utils/oscArgParser.ts

```diff
@@ -0,0 +1,27 @@
+import { ArgumentType } from 'node-osc';
+import { splitWhitespace } from 'ontime-utils';
+
+export function stringToOSCArgs(argsString: string | undefined): ArgumentType[] {
+  if (typeof argsString === 'undefined') {
+    return new Array<ArgumentType>();
+  }
+  const matches = splitWhitespace(argsString);
+
+  const parsedArguments = matches.map((argString: string) => {
+    const maybeNumber = Number(argString);
+    if (!Number.isNaN(maybeNumber)) {
+      //NOTE: number like: 1 2.0 33333
+      //TODO: we cant foce a float with our current osc lib
```

Comment:
This is not true. You removed all the code I had that forced types and gave us complete control over the OSC argument types.

Reply:
yeah sorry I totally broke your stuff in an attempt to fix something you already had working
I'm reverting some of it now

Reply:
No worries!